client/src/components/product/CategoryChip.tsx

```
@@ -0,0 +1,45 @@
+  return (
+    <button
+      onClick={onClick}
+      className={cn(
+        "group grid grid-cols-2 lg:grid-cols-5 justify-between items-center w-[40vw] md:w-44 gap-2 px-2 py-2 rounded-lg transition-all duration-300",
+        "border hover:border-emerald-600/50 hover:bg-emerald-50",
+        "focus:outline-none focus:ring-2 focus:ring-emerald-600/20",
+        isSelected
+          ? "border-emerald-600 bg-emerald-50"
+          : "border-gray-200 bg-white"
+      )}
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)